airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1527,8 +1527,8 @@ components:
         - status
       properties:
         status:
-          $ref: "#/components/schemas/JobStatus"
-    JobStatus:
+          $ref: "#/components/schemas/JobStatusReadStatus"
+    JobStatusReadStatus:
```

Comment:
this name is a mouthful, but needs to not collide with the more frequently used JobStatus